dependency-manager-tests/carthage/Makefile

```diff
@@ -1,7 +1,9 @@
-ifeq (${ci}, true)
-		GIT_BRANCH := ${BITRISE_GIT_BRANCH}
-else
-		GIT_BRANCH := $(shell git rev-parse --abbrev-ref HEAD)
+GIT_BRANCH := $(shell git rev-parse --abbrev-ref HEAD)
+ifneq (${BITRISE_GIT_BRANCH},)
+	GIT_BRANCH := ${BITRISE_GIT_BRANCH}
+endif
+ifneq (${BITRISE_GIT_TAG},)
+	GIT_BRANCH := ${BITRISE_GIT_TAG}
```

Comment:
`GIT_BRANCH` name is not totally accurate anymore but i couldn't find a better name

Reply:
nit: a "`GIT_REFERENCE`" could be more accurate.